ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -188,11 +203,15 @@
                   class="submitting-party"
                 >
                   <ContactInformation
-                    :contactInfo="getMhrTransferSubmittingParty"
+                    :contactInfo="isChangeLocationActive
+                      ? getMhrTransportPermit.submittingParty
+                      : getMhrTransferSubmittingParty"
                     :sectionNumber="1"
                     :content="submittingPartyChangeContent"
                     :validate="validateSubmittingParty"
-                    @setStoreProperty="setMhrTransferSubmittingParty"
+                    @setStoreProperty="isChangeLocationActive
+                      ? setMhrTransportPermit({ key: 'submittingParty', value: $event })
+                      : setMhrTransferSubmittingParty"
```

Comment:
And here

Reply:
As mentioned in chat, $event may need to pass explicitly because of the ternary?
The getter i commented on above is probably fine, since it populates the component